server/index.js

```diff
@@ -17,7 +17,7 @@ app.get('/', (req, res) => {
 });
 
 // Start the server
-const PORT = process.env.PORT || 8080;
+const PORT = process.env.PORT || 8000;
```

Comment:
why this change is required ?

Reply:
1. actually in my system port 8080 was not working so I changed this to 8000. But no worries I will resolve this.